lib/health-data-standards/models/entry.rb

```diff
@@ -25,12 +25,14 @@ class Entry
   field :oid, type: String
   field :reason, type: Hash
   field :health_record_field, type: Hash
-    
-  attr_protected :version
-  attr_protected :_id
-  attr_protected :created_at
-  attr_protected :updated_at
-  
+
+# attr_protected is from protected_attributes gem that is no longer maintained
```

Comment:
should we just remove this instead of commenting?

Reply:
i talked to brad about this. he recommended a comment. will remove if he prefers.

Reply:
I like leaving the comment in there because I have a feeling it's going to come back and haunt us later.  I would also just leave the original line in there, just commented out so we know how it was used.  (Although I guess we could look back in history.  Up to you).